src/main/daml/Daml/Finance/Interface/Util/Reference.daml

```diff
@@ -0,0 +1,72 @@
+-- Copyright (c) 2023 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE AllowAmbiguousTypes #-}
```

Comment:
Shall we rename this module `InterfaceKey`?

Reply:
I renamed it